virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -6,33 +6,25 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can attach virtual machines to multiple networks by using Linux bridges.
-You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
+{VirtProductName} is installed with a default internal pod network.
 
-To attach a virtual machine to an additional network:
+Depending on the type of networks you are using, it may be necessary to update your network node configuration.
 
-. Configure a Linux bridge.
-. Configure a bridge network attachment definition for a namespace in the web console or CLI.
-+
-[NOTE]
-====
-The network attachment definition must be in the same namespace as the pod or virtual machine.
-====
-. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
-** In the web console, create a NIC for a new or existing virtual machine.
-** In the CLI, include the network information in the virtual machine configuration.
+In order to use your virtual machines with external networks, you must create a Linux _network bridge_ to expose your network services outside the default internal pod network.
```

Comment:
I would move this to the beginning of the assembly. It is easy to miss here.

Reply:
From JF's comment it seemed, the node config might need to come first. So, another question. Yes, Linux is redundant ---- it's stated that way in the existing topics but I agree.